packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -252,6 +252,7 @@ export const CreateAlertDefinition = () => {
           />
           {openAddNotification && (
             <Drawer
+              data-qa-buttons="true"
```

Comment:
```suggestion
              data-qa-button
```
and consume it from here, we don't need true here also plural

Reply:
> and consume it from here, we don't need true here also plural

if we want use "ui.buttonGroup
.findButtonByTitle('Add notification channel')
.should('be.visible')
.click();," we have to keep data-qa-buttons="true" , implementation of findButtonByTitle: (buttonTitle: string): Cypress.Chainable => {
return cy
.get('[data-qa-buttons="true"]')
.findByText(buttonTitle)
.closest('button');
},
};